docs/content/2.deploy/providers/vercel.md

```diff
@@ -101,3 +101,39 @@ Instead, you have to use :
 ## Custom Build Output Configuration
 
 You can provide additional [build output configuration](https://vercel.com/docs/build-output-api/v3) using `vercel.config` key inside `nitro.config`. It will be merged with built-in auto generated config.
+
+## On-Demand Incremental Static Regeneration (ISR)
+
+On-demand revalidation allows you to purge the cache for an ISR route whenever you want, foregoing the time interval required with background revalidation.
+
+To revalidate a page on demand with Next.js:
```

Comment:
I think this can be removed. Nitro is not framework specific (nor Next.js is currently supported)

Reply:
ugh sry this is a mistake on my part. Will fix it shortly